scripts/qa_crowbarsetup.sh

```diff
@@ -2531,21 +2531,27 @@ function custom_configuration
             fi
             if [[ $want_ldap = 1 ]] ; then
                 local p="proposal_set_value keystone default"
-                if iscloudver 6plus; then
+                if iscloudver 7plus && grep -q domain_specific_config $pfile; then
+                    $p "['attributes']['keystone']['domain_specific_drivers']" true
+                    local l="['attributes']['keystone']['domain_specific_config']['ldap_users']['ldap']"
+                elif iscloudver 6; then
                     $p "['attributes']['keystone']['identity']['driver']" "'hybrid'"
                     $p "['attributes']['keystone']['assignment']['driver']" "'hybrid'"
+                    local l="['attributes']['keystone']['ldap']"
                 else
                     $p "['attributes']['keystone']['identity']['driver']" "'keystone.identity.backends.hybrid.Identity'"
                     $p "['attributes']['keystone']['assignment']['driver']" "'keystone.assignment.backends.hybrid.Assignment'"
+                    local l="['attributes']['keystone']['ldap']"
```

Comment:
maybe move this line above the `if` and only override in the 7plus case to reduce duplication